src/edu/csus/ecs/pc2/core/model/ContestInformation.java

```diff
@@ -79,6 +79,16 @@
     
     private int lastRunNumberSubmitted = 0;
     
+    /**
+     * Memory Limit for team's solution (application).
+     */
+    private int memoryLimitInMeg = 0;
+    
```

Comment:
ok, what value do you suggest?
The intent was that a zero value means null or not set, btw.   I would have used an Integer but trying to
not to avoid extra overhead.

Reply:
I literally answered your question in my comment:  `I think the default should be some reasonable value, such as 2048 (2GB)`